src/pages/welcome.tsx

```diff
@@ -52,6 +53,16 @@ const Container = styled.div`
   background-color: ${props => props.theme.colors.background};
 `
 
+const Subtext = styled.h4`
+  color: ${props => props.theme.colors.text};
+  font-size: calc(${props => props.theme.sizes.buttonText} * 0.75);
+  padding: 0 calc(${props => props.theme.sizes.buttonText} * 0.75);
```

Comment:
this could be expressed as 0.75em, no?
```suggestion
  padding: 0 0.75em;
```